packages/opentelemetry-browser-extension-autoinjection/package.json

```diff
@@ -65,18 +65,18 @@
     "@material-ui/core": "4.12.3",
     "@material-ui/icons": "4.11.2",
     "@material-ui/lab": "4.0.0-alpha.60",
-    "@opentelemetry/context-zone": "^0.24.0",
-    "@opentelemetry/core": "^0.24.0",
-    "@opentelemetry/exporter-collector": "^0.24.0",
-    "@opentelemetry/exporter-zipkin": "^0.24.0",
-    "@opentelemetry/instrumentation": "^0.24.0",
+    "@opentelemetry/context-zone": "0.24.0",
+    "@opentelemetry/core": "0.24.0",
+    "@opentelemetry/exporter-collector": "0.24.0",
+    "@opentelemetry/exporter-zipkin": "0.24.0",
+    "@opentelemetry/instrumentation": "0.24.0",
     "@opentelemetry/instrumentation-document-load": "^0.24.0",
```

Comment:
Why not also pinning this package?

Reply:
That looks to be the only dep in this PR which lives in this monorepo, so renovate doesn't touch it as it understands that lerna manages it.

Reply:
Do you think it would be good to manually change it to pinned? Or does that mess up lerna? I'm not really sure

Reply:
Yes, but

> Or does that mess up lerna?

I don't know the answer to this either

Either way it's a follow-up PR to this one.

Reply:
> Yes, but
> 
> > Or does that mess up lerna?
> 
> I don't know the answer to this either

It does not. Lerna actually has an `--exact` flag to get this behavior when you do `lerna add`

> Either way it's a follow-up PR to this one.

👍